addons/website/static/src/scss/website.backend.scss

```diff
@@ -218,11 +218,15 @@
 }
 
 .oe_stat_button {
-    &.o_stat_button_info:hover {
-        color: #666666 !important;
-        background-color: transparent !important;
-        opacity: 0.8 !important;
-        cursor: default !important;
+    &.o_stat_button_info {
+        pointer-events: none;
+
+        &:hover {
```

Comment:
@rdeodoo I must be missing something but this should not make sense: you disable pointer events on something and then add `:hover` rules on it -> those must be dead code

Reply:
Indeed, I added that one liner without paying much attention to the already existing lines.
Plus I did not actually know `pointer-events: none` was actually preventing hover effects.